redash/query_runner/databricks.py

```diff
@@ -0,0 +1,18 @@
+import base64
+from .hive_ds import Hive
+
+try:
+    from pyhive import hive
+    from thrift.transport import THttpClient
+    enabled = True
+except ImportError:
+    enabled = False
```

Comment:
You'll need to define an `enabled` class method to use this in the `DataBricks` query runner below to make use of this enabled variable.

```python
    @classmethod
    def enabled(cls):
        return enabled
```

Reply:
The `Hive` class has this method and the `DataBricks` class inherits from the `Hive` class so I don't think I have to re-define it.

Reply:
You'll still need to implement the method in the subclass since the `Hive` class is defined in the scope of the `hive_ds` module, which will make the enabled `Hive.enabled` classmethod use the module level variable from there, instead of a redefinition in a separate module. In other words, the subclass doesn't know about the scope of the module of the parent class. Here's an example to show: https://trinket.io/python/f5d33dff2e